mon-pix/app/services/oidc-identity-providers.js

```diff
@@ -14,6 +15,11 @@ export default class OidcIdentityProviders extends Service {
       .map((provider) => provider.organizationName);
   }
 
+  isIdentityProviderActivated(identityProviderSlug) {
```

Comment:
Le code de la fonction se base sur l'id du provider et non sur le slug, alors il faut faire correspondre l'argument de la méthode : 
```suggestion
  isIdentityProviderActivated(oidcProviderId) {
```